### PR TITLE
fix: fix tests

### PR DESCRIPTION
**Pull request by ArthurTriis1**
## What's the purpose of this pull request?

Fix tests

### CODE REVIEW

File: packages/core/cypress/integration/analytics.test.js [hunk 1]
```diff
@@ -347,7 +360,7 @@ describe('search event', () => {
       .click()
       .type('shirt')
       .within(() => {
-        cy.getById('fs-button')
+        cy.getById('store-input-mobile-button')
```
Comment (hellofanny) [inline packages/core/cypress/integration/analytics.test.js:363]: 🎗️ remember to change it when working on navbar / search

File: packages/components/src/molecules/SearchInputField/SearchInputField.tsx [hunk 2]
```diff
@@ -88,6 +88,7 @@ const SearchInputField = forwardRef<
         {...otherProps}
       />
       <IconButton
+        data-fs-icon-button="null"
```
Comment (hellofanny) [inline packages/components/src/molecules/SearchInputField/SearchInputField.tsx:91]: Do we really need this? 👀

File: packages/core/src/components/search/Filter/FilterSlider.tsx [hunk 3]
```diff
@@ -82,7 +82,7 @@ function FilterSlider({
           return (
             <UIFilterFacets
               key={`${testId}-${label}-${index}`}
-              testId={testId}
+              testId={`mobile-${testId}`}
```
Comment (eduardoformiga) [inline packages/core/src/components/search/Filter/FilterSlider.tsx:85]: Nice catch!

File: packages/core/package.json [hunk 4]
```diff
@@ -29,7 +29,7 @@
     "@envelop/graphql-jit": "^1.1.1",
     "@envelop/parser-cache": "^2.2.0",
     "@envelop/validation-cache": "^2.2.0",
-    "@faststore/api": "^2.0.97-alpha.0",
+    "@faststore/api": "*",
```
Comment (eduardoformiga) [inline packages/core/package.json:32]: Hmm, looks like it's getting the 1.12.38 version in `yarn.lock`. Shouldn't we use the evergreen version?

File: yarn.lock [hunk 5]
```diff
@@ -1407,6 +1407,18 @@
     minimatch "^3.0.4"
     strip-json-comments "^3.1.1"
 
+"@faststore/api@*":
+  version "1.12.38"
+  resolved "https://registry.yarnpkg.com/@faststore/api/-/api-1.12.38.tgz#1b244c908e78730257e3ca847efd22b3b5b8459d"
```
Comment (eduardoformiga) [inline yarn.lock:1412]: 👀

File: packages/core/package.json (hunk 4 above)
Comment (ArthurTriis1) [reply packages/core/package.json]: Yeah, it's wrong, I fixed

File: packages/components/src/molecules/SearchInputField/SearchInputField.tsx (hunk 2 above)
Comment (ArthurTriis1) [reply packages/components/src/molecules/SearchInputField/SearchInputField.tsx:91]: Yes 😞, without this the Search icon doesn't appear when we are searching I on mobile.

Without
Screenshot 2023-04-05 at 13 20 35

With
Screenshot 2023-04-05 at 13 21 10

**Review by hellofanny — APPROVED** [review]

File: packages/components/src/molecules/SearchInputField/SearchInputField.tsx (hunk 2 above)
Comment (hellofanny) [reply packages/components/src/molecules/SearchInputField/SearchInputField.tsx:91]: oh, I see.

File: packages/core/package.json [hunk 6]
```diff
@@ -29,7 +29,7 @@
     "@envelop/graphql-jit": "^1.1.1",
     "@envelop/parser-cache": "^2.2.0",
     "@envelop/validation-cache": "^2.2.0",
-    "@faststore/api": "^2.0.97-alpha.0",
+    "@faststore/api": "2.0.97-alpha.0",
```
Comment (eduardoformiga) [inline packages/core/package.json:32]:
```suggestion
    "@faststore/api": "^2.0.97-alpha.0",
```

**Review by eduardoformiga — APPROVED** [review]

**Pull request merged**